enclave/src/c_api/c_api.cc

```diff
@@ -16,6 +16,7 @@
 #include <vector>
 #include <string>
 #include <memory>
+#include "base64.h"
```

Comment:
Should this be `#include <dmlc/base64.h>`? I had to change it to that to build the code.

If so could you please move this line to under line 9, `#include <dmlc/thread_local.h>`? Thanks!